sr_demos/scripts/sr_demos/demo.py

```diff
@@ -154,345 +172,376 @@ def run(self):
-    elif hand_type == 'hand_extra_lite':
-        squeeze.update({f"{prefix}THJ5": hand_pos[f'{prefix}THJ5'] + offset2,
-                        f"{prefix}THJ2": hand_pos[f'{prefix}THJ2'] + offset2,
-                        f"{prefix}FFJ3": hand_pos[f'{prefix}FFJ3'] + offset2,
-                        f"{prefix}FFJ1": hand_pos[f'{prefix}FFJ1'] + offset2,
-                        f"{prefix}RFJ3": hand_pos[f'{prefix}RFJ3'] + offset2,
-                        f"{prefix}RFJ1": hand_pos[f'{prefix}RFJ1'] + offset2})
-    else:
-        squeeze.update({f"{prefix}THJ5": hand_pos[f'{prefix}THJ5'] + offset2,
-                        f"{prefix}THJ2": hand_pos[f'{prefix}THJ2'] + offset2,
-                        f"{prefix}FFJ3": hand_pos[f'{prefix}FFJ3'] + offset2,
-                        f"{prefix}FFJ1": hand_pos[f'{prefix}FFJ1'] + offset2,
-                        f"{prefix}MFJ3": hand_pos[f'{prefix}MFJ3'] + offset2,
-                        f"{prefix}MFJ1": hand_pos[f'{prefix}MFJ1'] + offset2,
-                        f"{prefix}RFJ3": hand_pos[f'{prefix}RFJ3'] + offset2,
-                        f"{prefix}RFJ1": hand_pos[f'{prefix}RFJ1'] + offset2,
-                        f"{prefix}LFJ3": hand_pos[f'{prefix}LFJ3'] + offset2,
-                        f"{prefix}LFJ1": hand_pos[f'{prefix}LFJ1'] + offset2})
-
-    # Squeeze object gently
-    hand_commander.move_to_joint_value_target_unsafe(squeeze, 0.5, wait=False, angle_degrees=True)
-    rospy.sleep(0.5)
-    hand_commander.move_to_joint_value_target_unsafe(hand_pos, 0.5, wait=False, angle_degrees=True)
-    rospy.sleep(0.5)
-    hand_commander.move_to_joint_value_target_unsafe(squeeze, 0.5, wait=False, angle_degrees=True)
-    rospy.sleep(0.5)
-    hand_commander.move_to_joint_value_target_unsafe(hand_pos, 2.0, wait=False, angle_degrees=True)
-    rospy.sleep(2.0)
-    execute_command_check(hand_commander, joint_states_config, 'pregrasp_pos', 2.0, 2.0)
-    execute_command_check(hand_commander, joint_states_config, 'start_pos', 2.0, 2.0)
-
-    rospy.loginfo("RF demo completed")
-
-
-def sequence_lf(hand_commander, joint_states_config, tactile_reading):
-    rospy.loginfo("LF demo started")
-
-    rospy.sleep(0.5)
-    # Initialize wake time
-    wake_time = time.time()
-    while True:
-        # For now, tactile_reading is only being considered for uni-manual
-        if tactile_reading is not None:
-            # Check if any of the tactile senors have been triggered
-            # If so, send the Hand to its start position
-            touched = tactile_reading.confirm_touched()
-            if touched is not None:
-                execute_command_check(hand_commander, joint_states_config, 'start_pos', 0.0, 2.0)
-                rospy.loginfo(f'{touched} touched!')
-                rospy.sleep(2.0)
-                if touched == "TH":
-                    break
+class Robot:
+    def __init__(self, robot_type, hand_type):
```

Comment:
Add pydoc please!

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b